include/ddsrouter/communication/Bridge.hpp

```diff
@@ -50,7 +50,7 @@ class Bridge
      * @param participant_database: Collection of Participants to manage communication
      * @param enable: Whether the Bridge should be initialized as enabled
      *
-     * In case any inside \c Track creation fails it wil throw a \c InitializationException
+     * In case any inside \c Track creation fails and will throw a \c InitializationException
```

Comment:
```suggestion
    * @throw \c InitializationException in case any inside \c Track creation fails.
```
Probably there is some @throw tag for Doxygen where this should be included

Reply:
Indeed :+1: